src/Shipping/ShippingController.php

```diff
@@ -60,6 +61,25 @@ function() {
 		add_filter( 'pre_update_option_pickup_location_pickup_locations', array( $this, 'flush_cache' ) );
 	}
 
+	/**
+	 * Gets a list of payment method ids that support the 'collectible' feature.
+	 *
+	 * @return string[] List of payment method ids that support the 'collectible' feature.
+	 */
+	public function get_collectible_method_ids() {
+		return array_keys(
```

Comment:
Is there a logic error here? 

`wp_list_pluck` would give you just the IDs from the filtered methods, but the indexes would be numeric. `array_keys` would just take those indexes right?

We could use `array_values`, or alternatively simplify this to use `array_map`—make the callback return either the method ID if its `collectible`, or return `false`. Then you can just run `array_filter` over the resulting mapped array.